examples/complete/stacks/orgs/cp/tenant1/dev/us-east-2-extras.yaml

```
@@ -0,0 +1,20 @@
+import:
+  - mixins/region/us-east-2
+  - orgs/cp/tenant1/dev/_defaults
+
+components:
+  terraform:
+    new-vpc:
```

Can we set this to `vpc/new`

```suggestion
    vpc/new:
```